ci/create-windows-image.ps1

```diff
@@ -14,8 +18,8 @@ $RESHARPER_CLI_VERSION="2018.1.2"
 $SPATIAL_VERSION="1.1.9"
 
 # From https://public-cdn.cloud.unity3d.com/hub/prod/releases-win32.json
-$UNITY_VERSION="2018.1.6f1"
-$UNITY_RELEASE_HASH="57cc34175ccf"
+$UNITY_VERSION="2018.2.0b10"
```

Comment:
@jessicafalk @zeroZshadow Once this is all rolled out, this change (plus a repin of the GDK to point to the new Windows image) will be all we need to do to support a new Unity release. :)

Reply:
I am SO looking forward to this :D